sources/packages/api/src/route-controllers/program-info-request/models/program-info-request.dto.ts

```diff
@@ -32,10 +32,18 @@ export interface ProgramInfoRequestAPIOutDTO {
   offeringDelivered: string;
   offeringType: OfferingTypes;
   offeringIntensity: OfferingIntensity;
+  courseDetails?: CourseDetails[];
   pirDenyReasonId?: number;
   otherReasonDesc?: string;
 }
 
+export interface CourseDetails {
```

Comment:
Can we change it to a class and also the `ProgramInfoRequestAPIOutDTO`. I saw the other interfaces, but at least these ones are PR-related-ish.